core/src/main/java/org/pentaho/di/core/database/DatabaseInterface.java

```diff
@@ -1200,4 +1200,11 @@ default String getUnsupportedTableOutputMessage() {
     return "";
   }
 
+  /**
+   * @return true if database supports the default char length validation
+   */
+  default boolean supportsGetLengthByChar() {
```

Comment:
Please change this method to: `lengthSemanticsUsesChars()`

Reply:
Fully disagree with this review comment. Please refer to:
"Although a method name can be any legal identifier, code conventions restrict method names. By convention, method names should be a verb in lowercase or **a multi-word name that begins with a verb in lowercase, followed by adjectives, nouns, etc. In multi-word names**, the first letter of each of the second and following words should be capitalized."
in https://docs.oracle.com/javase/tutorial/java/javaOO/methods.html

Reply:
I agree with Miguel on this one.

Reply:
@mbatchelor and @cravobranco, the only problem is that it's not a question of if the DB supports the lenght by bytes or chars but instead that the DB is configured to use it or not.

So If the issue here is that  the method doesn't start with a verb, may I suggest: `getLengthSemanticsUsesChars()`. I trust you'll agree with this one.